### DISCUSSION TITLE
Yi 1080p Home 6FUS - Hack 1.9.3.0

### DISCUSSION THREAD

**Discussion by baldus88** [indomus/forum#361] [Hardware & Firmware]
Ciao a tutti.
Chiedo a voi il supporto perchè non ne sto uscendo.
In poche parole sto cercando di modificare il firmware per la mia Yi 1080 dome con seriale 6FUS. 
Ho trovato 2 guide che sembrano includerla:
https://github.com/roleoroleo/yi-hack-MStar
https://github.com/TheCrypt0/yi-hack-v4
Ho seguito la guida, preso il firmware modificato, caricato sulla sd, riavviato la dome ma niente rimane tutto inviariato. 
La luce gialla lampeggia per pochi secondi, poi compare la blu ed è tutto come prima, l'app originale funziona correttamente.

Una cosa che non mi torna è la colonna firmware nella tabellina di roleoroleo, vedo scritto: 4.5.0, 4.2.0, 2.1.0 o anche unknown.
E' importante la release del firmware di partenza? Io dall'app vedo: 1.9.3.0E_201812141519
Chiaramente se provo a dare aggiorna firmware mi dice che ho già l'ultima.

Ho provato pure a vedere:
https://github.com/shadow-1/yi-hack-v3
Ma non ho trovato niente in merito al mio firmware.

Potete darmi supporto?

Fatemi sapere se vi servono ulteriori info.

Grazie
Saluti

**Comment by snakuzzo** [comment]
1. prova a cambiare SD
2. prova il firmware [allwinner](https://github.com/roleoroleo/yi-hack-Allwinner)

**Comment by baldus88** [comment]
Scrivo per cronaca magari può essere utile, ho trovato questo qua che fa al caso mio:
https://github.com/alienatedsec/yi-hack-v5/releases

Nella release avevo trovato proprio il mio firmware:
_yi_cloud_dome_1080p firmware base was 1.9.3.0D_201808301633 when first uploaded, which was problematic; especially when downgrading. I have uploaded the latest **1.9.3.0E_201812141519** firmware base. Try to reload the latest yi_cloud_dome_1080p_0.3.0.tgz package. Confirmed to be working on 9FUS and 6FUS models. You will need a quick SD card to make it happen._

Seguendo tutta la procedura è andata.

Grazie del supporto
